uiTests/src/test/resources/project.rhino.js

```diff
@@ -16,8 +16,13 @@ importClass(com.intellij.ui.GuiUtils);
 // Do not run any of the methods on the UI thread.
 function Project(underlyingProject) {
 
-  // Tab & model management
+  this.configure = function () {
+    const projectPropertiesComponent = PropertiesComponent.getInstance(underlyingProject);
+    projectPropertiesComponent.setValue('ASKED_ADD_EXTERNAL_FILES', true);
```

Comment:
Isn't the comment `// Let's disable VCS-related tooltips since they sometimes lead to an exception when closing the project.` still applicable here?